src/pages/login/login.tsx

```diff
@@ -1,20 +1,50 @@
-import React from 'react';
 import Button from '@mui/material/Button';
 import TextField from '@mui/material/TextField';
 import FormControlLabel from '@mui/material/FormControlLabel';
 import Checkbox from '@mui/material/Checkbox';
 import Box from '@mui/material/Box';
 import Typography from '@mui/material/Typography';
 import Container from '@mui/material/Container';
+import Alert from '@mui/material/Alert';
+import IconButton from '@mui/material/IconButton';
+import Collapse from '@mui/material/Collapse';
+import CloseIcon from '@mui/icons-material/Close';
+import axios from 'axios';
+import * as EmailValidator from 'email-validator';
+import React from 'react';
 
 export default function Login() {
-	const handleSubmit = (event : any) => {
+	const [open, setOpen] = React.useState(false);
+
+	async function verifyLogin(data: object) {
+
+		try {
+			const response = await axios.post('https://c80e5846-3798-4f42-8af7-230d2eb0a943.mock.pstmn.io/mock ', data);
```

Comment:
use the actual api endpoint fot the project. 

use `/api/${insert_your_route_here}`
and add a proxy inside of the viteconfig.ts